pkg/handler/api.go

```diff
@@ -183,15 +185,26 @@ func setCollectionResponseMetadata(collection api.CollectionMetadataSettable, c
 }
 
 func ParsePagination(c echo.Context) api.PaginationData {
-	pageData := api.PaginationData{Limit: DefaultLimit, Offset: DefaultOffset}
+	pageData := api.PaginationData{Limit: DefaultLimit, Offset: DefaultOffset, SortBy: DefaultSortBy}
 	err := echo.QueryParamsBinder(c).
 		Int("limit", &pageData.Limit).
 		Int("offset", &pageData.Offset).
+		String("sort_by", &pageData.SortBy).
 		BindError()
 
 	if err != nil {
 		log.Error().Err(err).Msg("Failed to bind pagination.")
 	}
+
+	if pageData.SortBy == DefaultSortBy {
+		err = c.Request().ParseForm()
+		if err != nil {
+			log.Error().Err(err).Msg("Failed to bind pagination.")
+		}
+		q := c.Request().Form
+		pageData.SortBy = strings.Join(q["sort_by[]"], ",")
+	}
```

Comment:
This turns the sort_by[] params in the url into a comma separated list for later consumption.